README.md

```diff
@@ -1,37 +1,25 @@
-# Docs
+# 🔖Docs
 
 [![Actions Status](https://github.com/woodlike/docs/workflows/CI/badge.svg)](https://github.com/woodlike/docs/actions)
 
-> Turn your Gatsby project into visual and interactive technical documentation.
+## Description
 
-## Motivation
+Docs is a Gatsby theme that programmatically generates visually polished and interactive technical documentation using MDX and Babel.
 
-### Development Requirements
+## 📚Documentation
 
-🚀 Node.js >=10
-🌲 Git
-🐈 yarn >= 1.12
+WIP
 
-### Getting started with development
+## 🗺 Roadmap
 
-```sh
-git clone git@github.com:woodlike/docs.git
-cd docs
-# Install packages
-yarn
-```
+[Top features](https://github.com/woodlike/docs/projects/1)
+[Top Bugs](https://github.com/woodlike/docs/issues?q=is%3Aissue+is%3Aopen+label%3Abug+sort%3Areactions-%2B1-desc+)
+[Newest Bugs](https://github.com/woodlike/docs/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
 
-### Commands
+## 🙏Support
 
-The list of most important commands to work with the selected workspace.
+[Discord](https://discord.gg/u2rucA)
```

Comment:
Es wäre gut hier eine weitere Alternative anzubieten, da nicht jeder Discord als eine Plattform nutzen möchte.

Reply:
First of all, thanks so much for the feedback @zoepage

I included  Slack to the *Support* section. Do you consider it to be an appropriate platform or do you have a better alternative?